apps/storybook/stories/ZIndex.tsx

```diff
@@ -0,0 +1,146 @@
+        gap={4}
+        dangerouslySetInlineStyle={{
+          __style: {
+            border: "1px solid black",
+          },
+        }}
+      >
+        <Box position="relative">
+          <Box
+            position="absolute"
+            width="100%"
+            height={1000}
+            dangerouslySetInlineStyle={{
+              __style: {
+                backgroundColor: "darkgray",
+                zIndex: currentZIndex,
+              },
+            }}
+          />
+        </Box>
+        <Box
+          width="100%"
+          height={64}
+          display="flex"
+          justifyContent="center"
+          alignItems="center"
+          dangerouslySetInlineStyle={{
+            __style: {
+              backgroundColor: "lightblue",
+              zIndex: getZIndex("sticky"),
+            },
+          }}
+        >
+          Theoretical sticky navbar (sticky layer)
+        </Box>
+
+        <Box display="flex" direction="column">
+          <Box
+            width="100%"
+            height={200}
+            display="flex"
+            justifyContent="center"
+            alignItems="center"
+            dangerouslySetInlineStyle={{
+              __style: {
+                backgroundColor: "lightgreen",
+                zIndex: getZIndex("base"),
+              },
+            }}
+          >
+            Theoretical main content (base layer)
+          </Box>
+
+          <Box
+            width="100%"
+            height={200}
+            display="flex"
+            justifyContent="center"
+            alignItems="center"
+            dangerouslySetInlineStyle={{
+              __style: {
+                backgroundColor: "#E6E6FA",
+                zIndex: getZIndex("menu"),
+              },
+            }}
+          >
+            Theoretical menu content (menu layer)
+          </Box>
+
+          <Box
+            width="100%"
+            height={200}
+            display="flex"
+            justifyContent="center"
+            alignItems="center"
+            dangerouslySetInlineStyle={{
+              __style: {
+                backgroundColor: "#FFB6C1",
+                zIndex: getZIndex("fixed"),
+              },
+            }}
+          >
+            Theoretical fixed content (fixed layer)
+          </Box>
+        </Box>
+      </Box>
+
+      {openModal && (
+        <Modal
+          header="Theoretical Modal"
+          onDismiss={() => setOpenModal(false)}
+          zIndex={getZIndex("modal")}
+        >
+          <Box
+            dangerouslySetInlineStyle={{
+              __style: {
+                backgroundColor: "#FFCC99",
+              },
+            }}
+          >
+            <Typography>Theoretical modal content (modal layer)</Typography>
+            <Typography>
+              tiny bug on this page, backdrop doesnt show because of no color
+              tokens but this is just used to demonstrate the layers!
+            </Typography>
+          </Box>
```

Comment:
@somethiiing Could we just add `import "@cambly/syntax-design-tokens/dist/css/variables.css";` to fix this bug?

### Before
![Screenshot 2024-10-04 at 6 25 34 AM](https://github.com/user-attachments/assets/1ea6b80f-d49c-4380-82e1-548eee279474)

### After
![Screenshot 2024-10-04 at 6 25 27 AM](https://github.com/user-attachments/assets/445ed4b3-5a74-4a6f-9837-9c84025d4d96)